release/.buildkite/build_pipeline.py

```diff
@@ -187,6 +187,19 @@ def __init__(self, name: str, retry: int = 0):
         SmokeTest("pbt"),
         # SmokeTest("serve"),
         # SmokeTest("serve_failure"),
+        # Full long running tests (1 day runtime)
+        "actor_deaths",
```

Comment:
Should we move many of these to core nightly?

Reply:
core nightly runs twice a day! So nightly must be fine. There's also core-daily, but I think it doesn't matter much.